migration-v1-v2.md

```diff
@@ -0,0 +1,226 @@
+
+```json
+{
+  "2022-12-19T00:00": {
+    "available_tickets": 100,
+    "variants": [
+      {
+        "id": "1",
+        "name": "Adult",
+        "available_tickets": 100,
+        "price": {
+          "face_value": "10",
+          "currency": "EUR"
+        }
+      },
+      {
+        "id": "2",
+        "name": "Child",
+        "available_tickets": 10
+      }
+    ]
+  }
+}
+```
+
+**New `price` Field to Describe Variants**:
+
+The schema of the response's `variants` field includes a new, _optional_ field called `price`. The supplier can use this 
+field to specify the price of an available variant. The schema of the `price` field is as follows:
+
+```json
+{
+  "face_value": string,
+  "currency": string
+}
+```
+
+For example, a valid value for the `variants` field could be:
+
+```json
+{
+  "variants": [
+      {
+        "id": "1",
+        "name": "Adult",
+        "available_tickets": 100,
+        "price": {
+          "face_value": "10.99",
+          "currency": "EUR"
+        }
+      }
+   ]
+}
+```
+
+### Reservation
+
+For details and examples please refer to the official 
+[Reservation Endpoint Documentation](https://tiqets.github.io/supplier-api/#tag/Availability/operation/getAvailability).
+
+#### Changes Affecting the Endpoint's Requests
+
+| Field          | Type of Change | v1             | v2           |
+| -----------    | -----------    | -----------    | -----------  |
+| `datetime`     | added          |  N/A           |  required    |
+| `timeslot_id`  | removed        |  `timeslot_id` |  N/A         |
+| `date`         | removed        |  `date`        |  N/A         |
+| `required_order_data`         | added        |  N/A        |  `required_order_data`        |
+| `required_visitor_data`         | added        |  N/A        |  `required_visitor_data`        |
+
+The lists of possible values for these fields are:
+
+- `required_order_data`: **PICKUP_LOCATION**, **DROPOFF_LOCATION**, **NATIONALITY**, **FLIGHT_NUMBER**, **PASSPORT_ID**
+- `required_visitor_data`: **FULL_NAME**, **EMAIL**, **PHONE**, **ADDRESS**, **PASSPORT_ID**, **DATE_OF_BIRTH**
+
+### Booking
+
+For details and examples please refer to the official 
+[Booking Endpoint Documentation](https://tiqets.github.io/supplier-api/#tag/Booking/operation/booking).
+
+#### Changes Affecting the Endpoint's Response
+
+| Field       | Type of Change | v1          | v2          |
+| ----------- | -----------    | ----------- | ----------- |
+| `barcode_position` | renamed    | `barcode_position`    | `barcode_scope`     |
+
+
+#### Changes Affecting the Endpoint's Requests
+
+The new version of the API supports an optional `HTTP` header named `TIQETS-TEST-ORDER`. Using this header the client 
+can make a booking request and flag the booking as a **test booking**. 
+
+**Important**: the API server **MUST** respond with a valid `HTTP` response and, **MUST** mark the booking as 
```

Comment:
This header was introduced for the testing tool so I would say that they **might** use this header to mark the test bookings because we don't have a way to check if the order was indeed marked.